### DISCUSSION TITLE
Is Baseball working?

### DISCUSSION THREAD

**Discussion by Brandon1212b** [vasqued2/ha-teamtracker#137] [Q&A]
My Orioles card is still showing no games available. Does this work for spring training or do I need to tag it to a different code for spring training?

**Comment by vasqued2** [comment]
Preseason can be a little wonky.  If there are more than 25 games over a two day span, only returns the first 25.  Others get dropped.  While this generally doesn’t happen in the regular season, it can happen in the preseason.

Also, as of 6:30am EST, the API is still returning yesterday’s games instead of today’s.  Since Baltimore didn’t play yesterday it is showing no games.  I don’t know when it flips over to today.  Once it flips, it should show.

Tampa bay is working for me and showing yesterdays results so the sensor itself is working.  I’ll try to take a look this evening and see what’s being returned by the API then to confirm the IN game sensor is also working.

**Comment by vasqued2** [comment]
The Indians  gpa e is working for me.  Lmk if you are still having issues

**Reply by Brandon1212b** [reply]
I didn't do anything but its working today, Not sure why yesterday was showing unavialble

**Comment by vasqued2** [comment]
There is a 25 game cap in the number of games returned by the ESPN API.  It generally returns the games from yesterday, today, and then maybe into the future until it hits the 25 game limit.  

During the preseason, with lots of double headers, there are times where more than 25 games are scheduled over a 2 day period.  In those situations, if you are lucky and your game is one of the first 25, it shows up.  If you are unlucky and you are 26 or higher, your game gets truncated.  The later your game, the more likely to be truncated.
